common/install.sh

```diff
@@ -113,17 +165,17 @@ if $FUNCTION; then
   else
     ui_print " "
     ui_print "   Enabling overlay features..."
-    sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
+    sed -i 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
```

Comment:
Do we want to do this with `prop_process`?

Reply:
prop_process doesn't work that way. The props have to be in a separate file from system.prop.